golang/internal/grpc/grpc.go

```diff
@@ -446,6 +466,38 @@ func executeSecretList(
 	}
 }
 
+func executeUpdate(ctx context.Context, command *agent.AgentUpdateRequest, updateFunc UpdateFunc) {
+	if updateFunc == nil {
+		log.Warn().Stack().Msg("gRPC self update is not implemented")
+		return
+	}
+
+	if err := updateFunc(ctx, command.Tag); err != nil {
```

Comment:
Do not mix one / multiple line errors.

Reply:
@nandor-magyar can we add this to linter somehow?

Reply:
I think it is fine, i could not find direct rule, there is a lib where we can define regex to rule out specific things.